<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQtClass</class>
 <widget class="QMainWindow" name="TestQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>691</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="listLayout">
     <item>
      <widget class="QListWidget" name="weatherList"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>359</y>
      <width>691</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="buttonLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="filterButton">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filterEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="computeButton">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="computeEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="resEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestQt.qrc"/>
 </resources>
 <connections/>
</ui>
